### PR TITLE
remove Precomputing Style

### PR DESCRIPTION
**Pull request by sunnylqm**
Since there are no precomputeStyle module now.

### CODE REVIEW

File: docs/DirectManipulation.md [hunk 1]
```diff
@@ -161,68 +161,6 @@ that call back to the `TouchableOpacity` component.
...
-            style={styles.container}>
-        <Text>Precompute style!</Text>
-      </View>
-    )
-  },
-});
-```
-[Run this example](https://rnplay.org/apps/8_mIAA)
-
-## setNativeProps to clear TextInput value
```
Comment (satya164) [inline docs/DirectManipulation.md]: May be this was removed in error? Can you add it back?

Comment (brentvatne) [inline docs/DirectManipulation.md]: :+1: @sunnylqm - can you add this back?

**Comment by sunnylqm** [comment]
Since there are no precomputeStyle module now, I can't see the point of this whole paragraph. And my real question is: What is changed to setNativeProps now? What should be noted except the gone module precomputeStyle?

**Comment by satya164** [comment]
@sunnylqm `setNativeProps` can be used as before. I'm not sure I understand your question :(

**Comment by sunnylqm** [comment]
@satya164  Isn't the removed paragraph a demonstration of using "precomputeStyle"(see the title) but not "setNativeProps"? Then the precomputeStyle module has gone, what should be specially demonstrated here?

**Comment by satya164** [comment]
@sunnylqm You've removed 2 paragraphs

1. Precomputing style
2. setNativeProps to clear TextInput value

The first one is gone. But the second one is unrelated.

**Comment by sunnylqm** [comment]
@satya164  Sorry! I remembered what I have done!  

There was a saying in the second paragraph:

    The `controlled` prop of TextInput can sometimes drop characters when the `bufferDelay` is low and the user types very quickly. 

And I didn't find anything about `controlled` or `bufferDelay` in either docs or sources. Therefore the following codes didn't make any sense if no one knows why. So maybe this paragraph is outdated, too?

**Comment by satya164** [comment]
@sunnylqm Controlled components are a basic concept in React - https://facebook.github.io/react/docs/forms.html#controlled-components

But anyways, this paragraph is not related to the `precomputeStyle` at all. Let's keep it as it is.

**Comment by sunnylqm** [comment]
@satya164   OK. I've reverted.

**Comment by will-** [comment]
@satya164 why not merge this commit to the master branch?
Got comfused with the `Precomputing style` paragraph today. It's still there. https://facebook.github.io/react-native/docs/direct-manipulation.html#precomputing-style

**Comment by satya164** [comment]
Wait, I merged into 0.15-stable? Sorry. @sunnylqm  please send pull requests only to master branch in future. Can you send this PR to master again?

**Comment by sunnylqm** [comment]
ok

**Comment by satya164** [comment]
@sunnylqm Hey, can you send the PR to master? I have reverted this PR on the `0.15-stable` branch.

**Comment by sunnylqm** [comment]
@satya164 https://github.com/facebook/react-native/pull/5066 Should I send this again?

**Comment by satya164** [comment]
@sunnylqm No. I hadn't seen that PR when I replied.

**Pull request merged**